addons/website_sale/tests/test_delivery_ui.py

```diff
@@ -72,3 +75,62 @@ def test_01_free_delivery_when_exceed_threshold(self):
         }])
 
         self.start_tour("/", 'check_free_delivery', login="admin")
+
+    def test_one_errored_carrier(self):
```

Comment:
to avoid copy-pasting code from the previous function you can extract it to `setUpClass` which will be executed before calling `test_` functions